Emby.Drawing/ImageProcessor.cs

```diff
@@ -378,26 +377,25 @@ private string GetCacheFilePath(string originalPath, ImageSize outputSize, int q
             return GetCachePath(ResizedImageCachePath, filename, "." + format.ToString().ToLower());
         }
 
-        public ImageSize GetImageSize(BaseItem item, ItemImageInfo info)
+        public ImageDimensions GetImageSize(BaseItem item, ItemImageInfo info)
```

Comment:
shouldn't we rename the property and the method to `GetImageDimensions` to match the return type?